content/2016.06.kinto-admin-1.0.md

```diff
@@ -0,0 +1,48 @@
+Title: Kinto-admin reaches 1.0
+Date: 2016-06-10
+Lang: en
+Url: kinto-admin-1.0
+Slug: kinto-admin-1.0
+Summary: Introducing kinto-admin, a Web UI for managing your Kinto data
+
+When choosing a database, it's a good idea to study a few important criteria: how nice is the API? Is it well documented? Will it scale? What is the tooling available? Tooling is especially important as it's what will make your life as a developer easier. Basically [kinto-admin] wants to be to Kinto what [phpMyAdmin] is to MySQL. We've mentioned kinto-admin a few times on this blog already, and today we're announcing the release of [version 1.0].
+
+![kinto-admin-logged-in]({filename}/images/kinto-admin-logged-in.png)
+
+Kinto-admin 1.0 allows users to discover and browse the buckets, collections and records they have access to on a given Kinto server. They can also create, update and delete them, provided they have the required permissions.
```

Comment:
It might be a good idea to explain a little bit about what the old versions were like and what we had to do to make this work.